Resources/Prototypes/Entities/Objects/Devices/Electronics/atmos_alarms.yml

```diff
@@ -0,0 +1,23 @@
+- type: entity
+  id: AirAlarmElectronics
+  parent: BaseItem
+  name: air alarm electronics
+  components:
+  - type: Tag
+    tags:
+    - AirAlarmElectronics
+  - type: Sprite
+    sprite: Objects/Misc/module.rsi
+    state: door_electronics
+
+- type: entity
+  id: FireAlarmElectronics
+  parent: BaseItem
+  name: fire alarm electronics
+  components:
+  - type: Tag
+    tags:
+    - FireAlarmElectronics
+  - type: Sprite
+    sprite: Objects/Misc/module.rsi
+    state: door_electronics
```

Comment:
i SWEAR these existed already but they might not

Reply:
don't think so?